### DISCUSSION TITLE
When should we expect new updates?

### DISCUSSION THREAD

**Discussion by Scudoxx** [Anjok07/ultimatevocalremovergui#748] [General]
Since the end of the MDX23 challenge in May, there have been many new and improved models. Some free services have already implemented them. Is there any information when UVR will release their updates?

**Comment by 0xdevalias** [comment]
From Discord:

- https://discord.com/channels/708579735583588363/785664354427076648/1126845712882225203

![image](https://github.com/Anjok07/ultimatevocalremovergui/assets/753891/163cf49c-cc32-40f3-821f-a5d4d39096bd)

**Reply by tandy-1000** [reply]
Can I get the invite link please?

**Reply by 0xdevalias** [reply]
I found the link in another issue on this repo when I searched "discord":

> Join the dedicated Discord server for all the info: https://discord.gg/zZ9CbEn5HE
> 
> _Originally posted by @thenormal in https://github.com/Anjok07/ultimatevocalremovergui/issues/344#issuecomment-1527894115_
